conf/layer.conf

```diff
@@ -14,6 +14,6 @@ LAYERVERSION_meta-python2 = "1"
 
 LAYERDEPENDS_meta-python2 = "core openembedded-layer"
 
-LAYERSERIES_COMPAT_meta-python2 = "hardknott"
+LAYERSERIES_COMPAT_meta-python2 = "hardknott gatesgarth"
```

Comment:
hmm I wonder if we really need it.  Can you try updating to 2021.03 release of yoe instead ?

Reply:
good point -- I did not realize hardknott was newer -- will try that.